sportsref/nfl/teams.py

```diff
@@ -240,6 +240,49 @@ def team_stats(self, year):
         return df.ix[df.player_id == 'Team Stats'].iloc[0]
 
     @sportsref.decorators.memoize
+    def injuries(self, year):
+        """ This gets the injuries from pro-football-reference for each team
+
+        :param year: year to query
+        :return: the pfrid and injury status per week for the year
+        """
+        # set link and table_name and then get the pyquery table
+        link = sportsref.nfl.BASE_URL + \
+               '/teams/{}/{}_injuries.htm'.format(self.teamID, str(year))
+        doc = pq(sportsref.utils.get_html(link))
```

Comment:
get_year_doc returns the html in pyquery form for the team and year combo. For example the link: 
"http://www.pro-football-reference.com/teams/rav/2012.htm"
The injury and snap count info are not on that page as far as I can tell. They are on related, but different links:
"http://www.pro-football-reference.com/teams/rav/2012_injuries.htm"
and 
"http://www.pro-football-reference.com/teams/rav/2012-snap-counts.htm"
These are entirely different html pages and content on those pages. Am I missing something? Is the data on the base page but hidden and still parsable?